<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DIsVerify</class>
 <widget class="QWidget" name="DIsVerify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: none;
background: none;
image: none;
background-color: rgba(0, 0, 0, 120);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>142</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>310</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="minimumSize">
      <size>
       <width>630</width>
       <height>510</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>630</width>
       <height>510</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 630px;
height: 510px;
background: #FFFFFF;
border-radius: 20px;</string>
     </property>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>630</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
image: none;

background: #22a4b3;

border-top-left-radius: 20px;
border-top-right-radius: 20px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_title">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>220</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;

width: 46px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #ffffff;
line-height: 34px;</string>
      </property>
      <property name="text">
       <string>再次确认</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_question">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>630</width>
        <height>231</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 190px;
height: 31px;
font-family: PingFang SC;
font-weight: bold;
font-size: 32px;
color: #000000;
line-height: 34px;
background: transparent;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_no">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>390</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1111</width>
        <height>1111</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(244, 96, 117, 255), stop:1 rgba(201, 67, 86, 255));
border-radius: 10px;
font-weight: bold;
color: white;
font-size: 32px;
font-family: PingFang SC;</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_yes">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>390</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1111</width>
        <height>1111</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #22A4B3;
border-radius: 10px;

font-family: PingFang SC;
font-weight: bold;
font-size: 32px;
color: #FFFFFF;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>309</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>142</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
